ansible/tasks/koala.yml

```diff
@@ -82,6 +82,15 @@
     - "systemctl daemon-reload"
     - "restart koala"
 
+- name: "ensure ruby is installed"
+  apt:
+    name: "ruby=1:{{ koala.ruby_version }}"
```

Comment:
We should take into account that this doesn't prevent automatic unattended upgrades of the package. If we would confine those to security updates, that's not that much of a problem, but I think the `apt` module doesn't allow downgrades. So this would fail if Ruby had had a security update between two deploys.

Reply:
Unattended upgrades has a blacklist. It might be good to put Ruby on there

Reply:
Having automatic security upgrades would be nice though. Koala itself hardcodes the Ruby version it requires, but Ubuntu only does backports so the _Ruby_ version would probably remain the same and therefore not conflict.